src/Fantomas.Tests/QuotationTests.fs

```diff
@@ -40,3 +40,36 @@ let logger =
         .Returns(())
         .Create()
 """
+
+[<Test>]
+let ``should format multiline quotation expressions idempotent, 2203`` () =
+    formatSourceString
+        false
+        """
+let action =
+    <@
+        let msg = %httpRequestMessageWithPayload
+        RuntimeHelpers.fillHeaders msg %heads
+        async {
+            let! response = (%this).HttpClient.SendAsync(msg) |> Async.AwaitTask
+            return response.EnsureSuccessStatusCode().Content
+        }
+    @>
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+let action =
+    <@ let msg = %httpRequestMessageWithPayload
```

Comment:
I can understand your reasoning behind the change.
However, this leads to [the following problem](https://fsprojects.github.io/fantomas-tools/#/?data=N4KABGBEAmCmBmBLAdrAzpAXFSAacUiaAYmolmPAIYA2as%2BEkAxgPZwWQA8AAgDoAnPsggQasAC5gAtmgDmYALxgApAAsJEgA4AlWAEcArugkBZdGipzYAdUQS1ABSoBPGqyrQwAem9gALGDinigKaFpUzOiCwqJgOobIEojSsAASsDRasAJoAHRINDQZnjloMvKqarCe5b4BQTXQoWDhkdFCIqJUaC7IzGDAPn6Bwc3IYRFRaDFdcWKSAIRgAuharMj0SmAAFCoORACUeWmaWgDCNIiwSXkAyjfQAIK9-TuycodgAD4AfGAvPrMPJPADuVHsABUegBrYZgAAcjRCE1aUw6sXmEFWEkMAhEq3CG3oeQAops8bA7oZmNM0HcJFRcWhzuxYDtjqykjcpPUkWMWm1prM4gBfeGjJqC9EzTo8X6zWb1J5FZHjSbtcqraQQkRUETSQw0ZK4uBgVjwMAOWBgFBwJIAfTIAC9YIrOrxGlIPtt1Gc9EYTOY0JZrHYHM43B5oCKIAkkil0plsrkCogiiU4LkKgp1E06n4AMxq6WasCxsA9IGDeHFgWooUYuZxcQSZaE9abG3KeoAdhLDZlFaxED2BzQx1O2ku11uD2Qz1ezHe8i%2B9QAjOuBxrhZ0R3E-gClyDwVDYfDN9u0Zrh6IcXiCWtibAyRTVtTaRYGUzDCy2Ry8i5CQeXhft6x3JsxTAeV3WEeo7lYVIr0bcoqFWKAQFFIA).

I won't deny that there are other places where Fantomas has this problem.
The point is that there are different ways of looking at this.

If it is ok with you I'd like you to bring it up with the F# style guide over at https://github.com/fsharp/fslang-design#style-guide? That would be interesting for you to see that process as well.
Once there is an outcome there, it can be picked up in the style guide and we can respect the outcome in Fantomas.

Reply:
Sure. To speed things up it might be good to include a proposal in the raised issue.
So if I understand you correctly, you prefer to have `<@`/`@>` on separate lines for multi line expressions, right?
Given your example, I think there is a good argument for that.

Reply:
Yes, I prefer it mainly because of the technical impact on Fantomas side.
Things are more straightforward if we can keep a multitude of the indent_size.
But I don't have the strongest opinion as I've never really used a quotation before.

Reply:
[Here](https://github.com/fsharp/fslang-design/issues/661) we go.